src/styles/blocks/card.css

```diff
@@ -0,0 +1,77 @@
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 16px;
+}
+
+.card__image-imac {
+  width: 160px;
+  height: 134px;
+  margin: 16px 3px 0;
+}
+
+.card__title {
+  margin-top: 36px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__product-code {
+  color: var(--color-text-secondary);
+  margin-top: 4px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__rating,
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 16px;
+}
+
+.card__rating-reviews {
+  font-weight: 400;
+  font-size: 10px;
+}
+
+.card__price {
+  margin-top: 24px;
+}
+
+.card__text-price {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-text-secondary);
+}
+
+.card__value-price {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__buttom-buy {
```

Comment:
There is a typo in the class name, it should be '.card__button-buy' instead of '.card__buttom-buy'.